scripts/test_serializer.py

```diff
@@ -0,0 +1,89 @@
+# This is a little script that I used for locally testing the serializer for
+# collections.
+# also see: https://www.django-rest-framework.org/tutorial/1-serialization/
+#
+#
```

Comment:
fine for development, before merging, this should be transformed into a regular test within the testing framework.

Reply:
You are right, I will make a regular test out of this

Reply:
Working on it. After discussion with Brice and Tobias we will merge this PR and I will open a new PR for the unit tests.
This way, Tobias can continue working on GeoDjango fields based on the develop branch with the most recent status of the serializers.

Reply:
:+1: